sdk/containerregistry/tests.yml

```diff
@@ -4,13 +4,14 @@ stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: containerregistry
+      SupportedClouds: 'Public,UsGov,China'
       Artifacts:
         - name: azure-containers-containerregistry
           groupId: com.azure
           safeName: azurecontainerscontainerregistry
       TimeoutInMinutes: 90
       EnvVars:
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_ID: $(CONTAINERREGISTRY_CLIENT_ID)
+        AZURE_CLIENT_SECRET: $(CONTAINERREGISTRY_CLIENT_SECRET)
+        AZURE_TENANT_ID: $(CONTAINERREGISTRY_TENANT_ID)
```

Comment:
Are these values being outputted by the resource deployment script?

Reply:
Yes.